QRHunter/app/src/main/java/com/example/qrhunter/data/repository/UserRepository.java

```diff
@@ -34,4 +48,35 @@ public void initializePlayer(Player player) {
             return;
         }
     }
+    private FirebaseFirestore db = FirebaseFirestore.getInstance();
```

Comment:
Is this still needed since DataRepository class already has a db instance?

Reply:
ur right its not needed